### ISSUE TITLE
[BUG] - Sorting shopping list items rarely takes effect

### ISSUE THREAD

**Issue by johnsonm** [mealie-recipes/mealie#2540]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/)

### What is the issue you are experiencing?

When I drag and drop shopping list items to put them in my store traversal order, sometimes they move into a random location, and more often they don't move at all.

I am deployed with docker on postgres following the deployment documentation. I pulled and redeployed latest images with ee121a12f8db, and reproduced this in mobile firefox, desktop firefox, and desktop chrome. No errors show in `docker logs mealie-api`

I can't try to replicate it in the demo environment because login is not working there with the default login and password.

### Steps to Reproduce

1. Create a shopping list
2. Drag items to new positions
3. See them return to their original position, or occasionally drop into some random other position.

### Please provide relevent logs

INFO:     172.24.0.4:41862 - "PUT /api/groups/shopping/items HTTP/1.1" 200 OK

### Mealie Version

ee121a12f8db

### Deployment

Docker (Linux)

### Additional Deployment Details

Standard postgres deployment

**Comment by michael-genson** [comment]
> I pulled and redeployed latest images with https://github.com/mealie-recipes/mealie/commit/ee121a12f8db33ecb4db5f8582f7ea9788d019e4

That's the latest "full" version, but it's about 11 months old by now. Can you reproduce on nightly?

> I can't try to replicate it in the demo environment because login is not working there with the default login and password

The demo uses a different username and password, try the login information in the banner at the top and see if you can reproduce. I'm unable to on the demo

**Comment by michael-genson** [comment]
If you're a regular shopping list user I recommend switching to nightly until there's a new version. There are tons of shopping list improvements in nightly

**Comment by johnsonm** [comment]
Oh. I didn't realize that beta was that old. I am new to the project and assumed it was relatively frequent. Thank you!

And yes, I used the username and password from the banner in the demo, or at least tried at least twice. I've logged in successfully in the demo before.

I'm now on nightly and confirm that the shopping list is working for me now.

Also now I will be set to pull the latest nightly before reporting other bugs.
